src/services/db/GitFileSystemService.ts

```diff
@@ -927,6 +927,7 @@ export default class GitFileSystemService {
                 sha,
                 path,
                 size: type === "dir" ? 0 : stats.size,
+                addedTime: stats.birthtimeMs,
```

Comment:
do we not need this for github?

Reply:
also, why not use `stats.mtimeMs` (last modified time - see [here](https://nodejs.org/docs/v9.9.0/api/fs.html#fs_stats_birthtimems))